nuxeo-core-binarymanager-jclouds/src/main/java/org/nuxeo/ecm/blob/jclouds/JCloudsBinaryManager.java

```diff
@@ -109,6 +113,8 @@ public void initialize(String blobProviderId, Map<String, String> properties) th
             throw new RuntimeException("Missing conf: " + BLOBSTORE_MAP_NAME_KEY);
         }
 
+        endpoint = Framework.getProperty(BLOBSTORE_ENDPOINT_KEY);
```

Comment:
At this point keeping compatibility is not a major issue as there are very few if any users of this code. This can be documented in the release notes, and you can have backward-compatibility using the current system if you think it's important. But reading config from the properties coming from the extension point is important for modularity.